### PR TITLE
Merge master 1.25.10 back into develop

### PR DESCRIPTION
**Pull request by Ferdev**
Merging 1.25.10 release changes back into develop branch.

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,13 +1,16 @@
 *** WooCommerce Shipping & Tax Changelog ***
 
-= 1.25.10 - 2021-xx-xx =
-* Tweak - Removes deprecated Jetpack constant JETPACK_MASTER_USER
```
Comment (frosso) [inline changelog.txt:4]: Looks like this went away? 👀 I believe it should be part of `.11`, since I don't see it in `master`

File: changelog.txt [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+= 1.25.11 - 2021-xx-xx =
 * Fix	  - Ensure status page is displayed on new WC navigation menu.
-* Add   - Add an endpoint for shipping label creation eligibility and share code for store eligibility.
```
Comment (frosso) [inline changelog.txt:6]: 👍

File: changelog.txt [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
 * Add   - Run phpcbf as a pre-commit rule.
 * Fix   - Fix PHPUnit tests. Rename `test_` to `test-` to match our phpcs rules. Remove travis and move to github action.
 * Tweak - Updated .nvmrc to use 10.16.0
 
+= 1.25.10 - 2021-03-24 =
+* Add   - Add an endpoint for shipping label creation eligibility and share code for store eligibility.
```
Comment (frosso) [inline changelog.txt:10]: 👍

File: changelog.txt (hunk 1 above)
Comment (Ferdev) [reply changelog.txt:4]: That was released as part of `1.25.9` https://github.com/Automattic/woocommerce-services/blob/f897309ea412a164be6460544f7e6dc9fc582edf/changelog.txt#L17 :smiley:

Comment (frosso) [reply changelog.txt:4]: Whoops, thanks! :D

**Review by frosso — APPROVED** [review]
💯

**Pull request merged**